File: base/cinema/.cache/draft/java/Adapter.java
public class Adapter {
  public Adapter(int capacidade) {
    sala = new Sala(capacidade);
  }

  public void reservar(String id, String fone, int ind) {
    sala.reservar(id, fone, ind);
  }

  public void cancelar(String id) {
    sala.cancelar(id);
  }

  public String toString() {
    return sala.toString();
    return "";
  }
}

File: base/cinema/.cache/draft/ts/adapter.ts
class Adapter {
    constructor(capacidade: number) {
        this.sala = new Sala(capacidade);
    }

    reservar(id: string, fone: number, ind: number): void {
        this.sala.reservar(id, fone, ind);
    }

    cancelar(id: string): void {
        this.sala.cancelar(id);
    }

    toString(): string {
        return this.sala.toString();
        return "";
    }

}

export {Adapter};